source/Manila/usage.rst

```diff
@@ -422,6 +422,79 @@ A share can be deleted by using the manila delete command:
                             Optional share group name or ID which contains the share (Experimental, Default=None).
 
 
+How to access a CephFS Share 
+-------------------------------
+
+There are two methods that can be used to mount a share onto a VM:
+
+- Kernel method 
+  
+- Ceph FUSE method 
+
```

Comment:
`The kernel method is preferred as it uses significantly less CPU marshalling data through userspace`